itk_dev_shared_components/kmd_nova/nova_tasks.py

```diff
@@ -89,8 +89,8 @@ def get_tasks(case_uuid: str, nova_access: NovaAccess, limit: int = 100) -> list
             uuid = task_dict['taskUuid'],
             title = task_dict['taskTitle'],
             description = task_dict.get('taskDescription'),
-            case_worker_ident = task_dict['caseWorker']['ident'],
-            case_worker_uuid = task_dict['caseWorker']['id'],
+            case_worker_ident = task_dict['caseWorker']['ident'] if 'caseWorker' in task_dict else None,
+            case_worker_uuid = task_dict['caseWorker']['id'] if 'caseWorker' in task_dict else None,
```

Comment:
Adding a `case_worker` variable would make this a little cleaner, e.g.

```py
case_worker = task_dict['caseWorker'] if 'caseWorker' in task_dict else None
task = Task(
  …,
  case_worker_ident = case_worker['ident'] if case_worker else None,
  case_worker_uuid = case_worker['uuid'] if case_worker else None,
  …
)
```

Reply:
I actually started with this but I didn't really think it was that much cleaner